packages/earth-admin/src/pages-client/places/dashboard.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
```

Comment:
hmm i would rename this to something else, PlacesDashboard or something? PageDashboard? just looking at the file name i thought it was the dashboards section